autogpts/autogpt/pyproject.toml

```diff
@@ -22,16 +22,17 @@ serve = "autogpt.app.cli:serve"
 
 [tool.poetry.dependencies]
 python = "^3.10"
-# autogpt-forge = { path = "../forge" }
-autogpt-forge = {git = "https://github.com/Significant-Gravitas/AutoGPT.git", subdirectory = "autogpts/forge"}
+#TODO: Add this to the documentation so people can use the local or remote version. E.g. setting to local was the only way to fix the duck duck go dependency issue.
+autogpt-forge = { path = "../forge" }
+#autogpt-forge = {git = "https://github.com/Significant-Gravitas/AutoGPT.git", subdirectory = "autogpts/forge"}
```

Comment:
This will break the Docker build.

Reply:
What part will break it? Just updating duckduck go version?

Reply:
Or maybe the Dockerfile file can account for it?